analysis/microbiomeModelingToolbox/tutorial_mgPipe.m

```diff
@@ -35,12 +35,16 @@
 %%
 % path to where the COBRA Toolbox is located
 global CBTDIR
-
-% path to microbe models (download AGORA models from https://www.vmh.life/#downloadview)
-modPath = [getenv('HOME') filesep 'AGORA' filesep 'mat'];
+cd([CBTDIR filesep 'tutorials' filesep 'analysis' filesep 'microbiomeModelingToolbox'])
+% We will use the AGORA resource (Magnusdottir et al., Nat Biotechnol. 2017 
+% Jan;35(1):81-89) in this tutorial. AGORA version 1.02 is available at
+% www.vmh.life. Download AGORA and place the models into a folder.
+system('curl -O https://www.vmh.life/files/reconstructions/AGORA/1.02/Agora-1.02.zip')
+unzip('Agora-1.02.zip','AGORA')
+modPath=[CBTDIR filesep 'tutorials' filesep 'analysis' filesep 'microbiomeModelingToolbox' filesep 'AGORA' filesep 'mat'];
 
 % path where to save results
-resPath = [CBTDIR filesep '.tmp']
+resPath = [CBTDIR filesep '.tmp'];
```

Comment:
Could this not also be a `.tmp` folder within the tutorial folder?

Reply:
I was thinking about the `.tmp` folder at the root, but as these are files in a tutorial, I prefer that the user sees them next to the tutorial, and not in a hidden folder at the root.